### PR TITLE
Themes (single-site-jetpack): Highlight active theme in WPCOM themes list

### PR DESCRIPTION
**Pull request by ockham**
This additional `suffixThemeId()` wrapped around individual selectors' arguments inside `ThemesSelection`'s `connect()` call is kind of a gross hack (and only relevant to the WPCOM list in single-site-jetpack mode), but I'm not sure if we can do a lot better as long as we have to work with those `-wpcom` suffices. Maybe we can encapsulate/contain it a bit more. 

Maybe use a `suffix` string prop instead of `queryWpcom`, default to `''`, and always append that prop (so we get at least rid of the ternary/`queryWpcom` handling logic)?

Any ideas @budzanowski @seear?

### CODE REVIEW

File: client/my-sites/themes/single-site-jetpack.jsx [hunk 1]
```diff
@@ -114,6 +113,7 @@ export default connectOptions(
 									return getScreenshotOption( theme ).label;
 								} }
 								trackScrollPage={ props.trackScrollPage }
+								queryWpcom= { true }
```
Comment (seear) [inline client/my-sites/themes/single-site-jetpack.jsx]: extra space

**Comment by seear** [comment]
A different approach might be to use different `isThemeActive` and `isThemePurchased` selectors for the wpcom list in single-site-jetpack mode. Not sure if the current component arrangement allows us to do this?

**Comment by ockham** [comment]
> A different approach might be to use different `isThemeActive` and `isThemePurchased` selectors for the wpcom list in single-site-jetpack mode. Not sure if the current component arrangement allows us to do this?

Hmm. That'd still amount to changing `ThemesSelection` for the suffix use case, and kind of increases the surface of custom hacks. Not liking it too much I think 😕 At least those custom selectors shouldn't live in `state/themes/selectors` but only locally inside `themes-selection`...

Oh well. Maybe the ugliness is inherent in having that suffix at all. To unblock us (e.g. installation state visual feedback), I'm inclined to merge and iterate, if you don't mind. I might try this however:

> Maybe use a `suffix` string prop instead of `queryWpcom`, default to `''`, and always append that prop (so we get at least rid of the ternary/`queryWpcom` handling logic)?

**Comment by ockham** [comment]
Bit of a brainstorm for a possible future refactor:

* Ideally we'd centralize at least both this hack and the [adding of the `-wpcom` suffix](https://github.com/Automattic/wp-calypso/blob/fd79e515c5c365e7b3c426161ce88c506b76c213/client/my-sites/themes/theme-options.js#L63) we're currently doing inside `theme-options`.
* Maybe this means suffix addition should happen inside `themes-list`? Seems even more low-level.

**Pull request merged**